packages/asset-buyer/src/asset_buyer.ts

```diff
@@ -271,8 +271,6 @@ export class AssetBuyer {
                 orders,
                 assetBuyAmount,
                 orders.map(o => o.signature),
-                feeOrders,
```

Comment:
The new forwarder doesn't require these params?

Reply:
Yeah! 🤷‍♀ Removed everywhere.